src/main/kotlin/pt/up/fe/ni/website/backend/service/ErrorMessages.kt

```diff
@@ -0,0 +1,35 @@
+package pt.up.fe.ni.website.backend.service
+
+object ErrorMessages {
+    fun postNotFound(postId: Long): String {
+        return "post not found with id $postId"
+    }
+
+    fun projectNotFound(id: Long): String {
+        return "project not found with id $id"
+    }
+
+    fun emailAlreadyExists(): String {
+        return "email already exists"
+    }
+
+    fun accountNotFound(id: Long): String {
+        return "account not found with id $id"
+    }
+
+    fun emailNotFound(email: String): String {
+        return "account not found with email $email";
+    }
+
+    fun invalidCredentials(): String {
+        return "invalid credentials"
+    }
+
+    fun invalidRefreshToken(): String {
+        return "invalid refresh token"
+    }
+
+    fun expiredRefreshToken(): String {
+        return "refresh token has expired"
+    }
```

Comment:
```suggestion
    const val emailAlreadyExists = "email already exists"

    const val invalidCredentials = "invalid credentials"

    const val invalidRefreshToke = "invalid refresh token"

    const val expiredRefreshToken = "refresh token has expired"

    fun postNotFound(postId: Long) = "post not found with id $postId"

    fun projectNotFound(id: Long) = "project not found with id $id"

    fun accountNotFound(id: Long) = "account not found with id $id"

    fun emailNotFound(email: String) = "account not found with email $email"
```

My second suggestion is to change the constant error messages to `const val` instead of functions, since they always return a static value. But I'll leave that decision up to you